src/cr-logging.eventstore/EventStoreSerilogLogger.cs

```diff
@@ -0,0 +1,69 @@
+﻿// <copyright file="EventStoreSerilogLogger.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.Logging.EventStore
+{
+    using System;
+    using global::EventStore.ClientAPI;
+    using Serilog.Core;
+
+    /// <summary>
+    /// Extension to support logging of EventStore events.
```

Comment:
Not really an extension, would be better to describe it as a wrapper around a Serilog logger to support logging ES Events.